docs/ai-consumers/invoke-service-snet-cli.md

```diff
@@ -30,11 +30,11 @@ See the [CLI documentation](http://snet-cli-docs.singularitynet.io/) for full de
 Ether is used to pay for interactions on the block chain (known as gas).
 
 The transactions you make a call to SingularityNET are:
-- Transfer AGI into the multi-party escrow account,
+- Transfer AGIX into the multi-party escrow account,
 - Create a payment channel for a service published in the SingularityNET registry, and
-- Transfer AGI into the payment channel and set the timeout
+- Transfer AGIX into the payment channel and set the timeout
 
-After that, you interact with the service directly and won't need to pay for further transactions unless you want add more AGI
+After that, you interact with the service directly and won't need to pay for further transactions unless you want add more AGIX
```

Comment:
"unless you want add more AGIX" should be "unless you want to add more AGIX"

Reply:
@raamb  Resolved